.github/workflows/os-release-issues.yml

```diff
@@ -97,7 +97,7 @@ jobs:
           # Read the file contents and replace the placeholders
           file_path="../opensearch-build/.github/ISSUE_TEMPLATE/component_release_template.md"
           RELEASE_VERSION="${{ matrix.release_version }}"
-          RELEASE_ISSUE_NUMBER=$(echo '${{ steps.check_if_build_repo_issue_exists.outputs.issues }}' | jq '.[0].number')
+          RELEASE_ISSUE_NUMBER=$(echo '${{ steps.check_if_build_repo_issue_exists.outputs.issueNumbers }}' | jq '.[0]')
```

Comment:
I dont see `outputs` defined anywhere. The returns do not have `issueNumbers` either https://github.com/actions-cool/issues-helper/tree/v3#find-issues

Reply:
Ya Sayali
```
[📝 AC] [Init] [opensearch-project/opensearch-build => undefined]
[📝 AC] [doQueryIssues] issueNumbers is ---> [3998]
[📝 AC] [doFindIssues] issues --> [{"auth":"github-actions[bot]","number":3998,"title":"[RELEASE] Release version 2.11.0","body":"## Release OpenSearch and OpenSearch Dashboards 2.11.0\n\nI noticed that a manifest was automatically created in [manifests/2.11.0](/opensearch-project/opensearch-build/tree/main/manifests/2.11.0). Please follow the following checklist to make a release.\n\n<details><summary>How to use this issue</summary>\n<p>\n\n## This Release Issue\n\nThis issue captures the state of the OpenSearch release, its assignee (Release Manager) is responsible for driving the release. Please contact them or @mention them on this issue for help. There are linked issues on components of the release where individual components can be tracked. For more information check the the [Release Process OpenSearch Guide](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md).\n\n</p>\n</details>\n\nPlease refer to the following link for the release version dates: [Release Schedule and Maintenance Policy](https://opensearch.org/releases.html).\n\n### [Preparation](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#preparation)\n\n- [ ] [Release manager](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-manager) assigned.\n- [ ] Existence of label in each component repo. For more information check the [release-label](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-label) section.\n- [ ] [Increase the build frequency](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#increase-the-build-frequency).\n- [ ] [Release Issue](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-issue).\n\n### [Campaigns](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#campaigns)\n\n- [ ] [Component Release Issue](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#component-release-issue).\n- [ ] [Release Campaigns](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-campaigns).\n\n### [Release Branch and Version Increment](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-branch-readiness) - _Ends __REPLACE_RELEASE-minus-4-days__\n\n- [ ] [Core Release Branch](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#core).\n- [ ] [Core Version Increment](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#core-version-increment).\n- [ ] [Components Release Branch](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#components).\n- [ ] [Components Version Increment](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#components-version-increment).\n\n### [Feature Freeze](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#code-complete-and-feature-freeze) - _Ends __REPLACE_RELEASE-minus-12-days__\n\n- [ ] OpenSearch / OpenSearch-Dashboards core and components teams finalize their features.\n\n### [Code Complete](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#code-complete-and-feature-freeze) - _Ends __REPLACE_RELEASE-minus-[10](https://github.com/opensearch-project/opensearch-build/actions/runs/6153702599/job/16697978174#step:4:11)-days___\n\n- [ ] Mark this as done once the [Code Complete](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#code-complete-and-feature-freeze) is reviewed.\n- [ ] Create/Verify pull requests to add each component to relase input [manifests/2.[11](https://github.com/opensearch-project/opensearch-build/actions/runs/6153702599/job/16697978174#step:4:12).0/opensearch-2.11.0.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.11.0/opensearch-2.11.0.yml) and [manifests/2.11.0/opensearch-dashboards-2.11.0.yml](/opensearch-project/opensearch-build/tree/main/manifests/2.11.0/opensearch-dashboards-2.11.0.yml).\n\n### [Release Candidate Creation and Testing](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-candidate-creation-and-testing) - _Ends __REPLACE_RELEASE-minus-6-days___\n\n- [ ] [Generate Release Candidate](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-candidate).\n- [ ] [Integ Test TAR](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#integ-test-tar).\n- [ ] [Integ Test RPM](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#integ-test-rpm).\n- [ ] [Docker Build and Scan](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#docker-build-and-scan).\n- [ ] [Backwards Compatibility Tests](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#backwards-compatibility-tests).\n- [ ] [Windows Integration Test](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#windows-integration-test).\n- [ ] [Broadcast and Communication](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#broadcast-and-communication).\n- [ ] [Release Candidate Lock](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-candidate-lock).\n\n### [Performance testing validation](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#benchmark-tests) - _Ends __REPLACE_RELEASE-minus-6-days___\n\n- [ ] Post the benchmark-tests\n- [ ] Longevity tests do not show any issues.\n\n### [Pre Release](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#pre-release) - _Ends __REPLACE_RELEASE-minus-1-days___\n\n- [ ] [Release Labeled Issues](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-labeled-issues).\n- [ ] [Go or No-Go](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#go-or-no-go).\n- [ ] [Promote Repos](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#promote-repos).\n- [ ] [Promote artifacts](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#promote-artifacts).\n- [ ] [Release Notes](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-notes).\n\n### [Release](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#main-release) - _Ends {__REPLACE_RELEASE-day}_\n\n- [ ] [Maven Promotion](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#maven-promotion).\n- [ ] [Docker Promotion](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#docker-promotion).\n- [ ] [Release Validation](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-validation).\n- [ ] [Collaboration with the Project Management Team](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#collaboration-with-the-project-management-team).\n\n### [Release Checklist](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-checklist).\n\n<br>\n<details><summary>Release Checklist</summary>\n<p>\n\n### Pre-Release activities\n- [ ] Promote Repos.\n   - - [ ] OS\n   - - [ ] OSD\n- [ ] Promote Artifacts.\n   - - [ ] Windows\n   - - [ ] Linux Debian\n   - - [ ] Linux RPM\n   - - [ ] Linux TAR\n- [ ] Consolidated Release Notes.\n\n### Release activities\n- [ ] Docker Promotion.\n- [ ] Release Validation part 1.\n     -  - [ ] OpenSearch and OpenSearch Dashboard Validation.\n     -  - [ ] Validate the native plugin installation.\n- [ ] Merge consolidated release notes PR.\n- [ ] Website and Documentation Changes.\n    - - [ ] Merge staging website PR.\n    - - [ ] Promote the website changes to prod.\n    - - [ ] Add website alert.\n- [ ] Release Validation part 2.\n    -  - [ ] Validate the artifact download URL's and signatures. \n- [ ] Release Validation part 3.\n    -  - [ ] Trigger the validation build (Search for `Completed validation for <>` in the logs).\n- [ ] Maven Promotion.\n- [ ] Publish blog posts.\n- [ ] Advertise on Social Media.\n- [ ]  Post on public slack and Github Release issue.\n\n### Post-Release activities\n- [ ] Release Tags.\n- [ ] Input Manifest Update.\n- [ ] Decrease the Build Frequency.\n- [ ] OpenSearch Build Release notes.\n- [ ] Retrospective Issue.\n- [ ] Helm and Ansible Playbook release.\n- [ ] Upcoming Release Preparation.\n\n</p>\n</details>\n<br>\n\n### [Post Release](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#post-release)\n\n- [ ] [Release Tags](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#release-tags).\n- [ ] [Input Manifest Update](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#input-manifest-update).\n- [ ] [OpenSearch Build Release notes](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#opensearch-build-release-notes).\n- [ ] [Decrease the Build Frequency](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#decrease-the-build-frequency).\n- [ ] [Retrospective Issue](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#retrospective-issue).\n- [ ] [Helm and Ansible Playbook release](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#helm-and-ansible-playbook-release).\n- [ ] [Upcoming Release Preparation](https://github.com/opensearch-project/opensearch-build/blob/main/RELEASE_PROCESS_OPENSEARCH.md#upcoming-release-preparation).\n\n### Components\n\n__Replace with links to all component tracking issues.__\n\n| Component | On track | Release Notes |\n| --------- | -------- | ----- |\n| {COMPONENT_ISSUE_LINK} | {INDICATOR}} | {STATUS} |\n\n<details><summary>Legend</summary>\n<p>\n\n| Symbol | Meaning |\n| -------- | ---------- |\n| :green_circle: | On track with overall release |\n| :yellow_circle: | Missed last milestone |\n| :red_circle: | Missed multiple milestones |\n\n</p>\n</details>","state":"open","created":"2023-09-12T00:16:16Z","updated":"2023-09-12T00:16:17Z"}]
```
But this is part of the debug log, wanted to try with `issueNumbers`.